keps/sig-instrumentation/1753-logs-sanitization/README.md

```diff
@@ -198,6 +204,16 @@ To allow configuring if logs should be sanitized we will introduce a new logging
 
 ### Test Plan
 
+Tests should cover two things:
+* Test feature log sanitization works e2e
+* Test Kubernetes components don't leak sensitive data
```

Comment:
What about any potential performance degradations?

Reply:
I don't think we should add specific mechanism to monitor performance degradation and can rely on existing qualification done by scalability team. 

Before enabling this feature by default we will work with SIG Scalability on qualifying (e.g. before major changes for Structured Logging we requested to run scalability tests on PR before merging)

Before feature is enabled by default, performance should not be big concern we should just ensure that e2e periodic test doesn't fail due to overhead

Reply:
It's not a matter of just "not failing" - we shouldn't eat all the slack we have with that. It's a matter of overhead.

But I agree that I wouldn't add a dedicated stuff for it per-se.